HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -67,8 +67,61 @@ Greatest common divisor
 
 method:: round
 Round to multiple of aNumber
+
 method:: roundUp
-round up to a multiply of aNumber
+Round up to a multiply of aNumber. For roundDown use Link::Classes/SimpleNumber#-trunc#trunc::.
+
+method:: trunc
+Truncate to multiple of aNumber
+
+method:: softRound
+Rounds the value to a multiple of strong::resolution::. By using strong::tolerance:: and strong::strength:: you can control which values will be rounded, and by how much.
+
+Conceptually this is the equivalent of MIDI quantization in a DAW/MIDI sequencer. In particular it allows a certain sloppiness close to the strong::resolution:: value.
+argument::resolution
+Round this value to a multiple of resolution. E.g. if you chose 1, then all values would be rounded to the nearest integer.
+argument::tolerance
+Values that are within +-strong::tolerance:: from a multiple of strong::resolution:: will be left as they are.
+
+E.g. if you chose a resolution value of 0.5 and a tolerance of 0.01, then the values 0.501 and 0.499 would be left as they are, but the value 0.502 would become 0.5.
+argument::strength
+Determines the degree to which this number will be changed.
+
+If strength is 1, then this function will return the nearest resolution. If it is 0, then value of this number will be left unchanged.
+
+E.g. If the resolution was 1 and strength was 0.5, then the value 0.6 would become 0.8.
+discussion::
+code::
+((0..10) / 5).collect { |num| [num, num.softRound(1, 0, 1)] };
+((0..10) / 5).collect { |num| [num, num.softRound(1, 0.3, 1)] };
+((0..10) / 5).collect { |num| [num, num.softRound(1, 0, 0.5)] };
+::
+
+method:: snap
+Rounds the values strong::tolerance:: distance from resolution to a multiple of resolution. By using strong::tolerance:: and strong::strength:: you can control when values will be rounded, and by how much.
+
+Conceptually this is the equivalent of 'snap' in a graphics program. Values within a certain distance (strong::tolerance::) from a grid line are snapped to it. All other values are unchanged.
+
+argument::resolution
+Snap this value to a multiple of resolution. E.g. if you chose 1, then all values would be rounded to the nearest integer.
+
+argument::tolerance
+Only values that are greater +-strong::tolerance:: from a multiple of strong::resolution:: value will be changed. Values that are less than strong::tolerance:: will be unchanged.
```

Comment:
fyi, documentation is UTF-8 encoded, so you could use ± if you wanted to.

Reply:
Ha! Done.